lib/luwra/usertypes.hpp

```diff
@@ -217,7 +218,7 @@ struct Value<U*> {
 	 * \returns Number of values that have been pushed
 	 */
 	static inline
-	size_t push(State* state, const T* ptr) {
+	size_t push(State* state, T* ptr) {
```

Comment:
Why is remove the qualification? Every type `T` can be implicitly converted to `const T`, but not the other way around.